.github/workflows/maven_test.yml

```diff
@@ -172,7 +177,7 @@ jobs:
         if: contains(matrix.modules, 'resource-managers#yarn') || (contains(matrix.modules, 'sql#core')) || contains(matrix.modules, 'connect')
         with:
           python-version: '3.11'
-          architecture: x64
+          architecture: ${{ inputs.architecture }}
```

Comment:
The installation of Python requires specifying the arm64 architecture; otherwise, the installation will fail.

https://github.com/LuciferYang/spark/actions/runs/14371131848/job/40294266753

![image](https://github.com/user-attachments/assets/77263fc0-b018-4169-8375-266155d79113)

Reply:
If this change needs to be split into a separate pull request, please let me know.